provisioner.go

```diff
@@ -169,8 +174,10 @@ func (p *defaultProvisioner) provisionAstilectron(ctx context.Context, d Dispatc
 }
 
 // provisionElectron provisions electron
-func (p *defaultProvisioner) provisionElectron(ctx context.Context, d Dispatcher, paths Paths, s ProvisionStatus, appName, os string) error {
-	return p.provisionPackage(ctx, d, paths, s.Electron, p.moverElectron, "Electron", VersionElectron, paths.ElectronUnzipSrc(), paths.ElectronDirectory(), func() (err error) {
+func (p *defaultProvisioner) provisionElectron(ctx context.Context, d Dispatcher, paths Paths, s ProvisionStatus, appName, os, arch string) error {
+	electron := s.Electron[fmt.Sprintf("%s-%s", os, arch)]
```

Comment:
Could you avoid creating a new variable here and inject `s.Electron[fmt.Sprintf("%s-%s", os, arch)]` directly in `p.provisionPackage` ?